### DISCUSSION TITLE
How to set custom openai api as default

### DISCUSSION THREAD

**Discussion by edge-sky** [ItzCrazyKns/Vane#422] [Q&A]
I deploy Perplexica in my server which without GPU, so I use custom openai api. For multiple device use, I need to set the **key** and **base_url** in every brower. So is it have any way to set default api and key in the server like "OpenAI API Key" and "Ollama API URL"?
By the way, I not see any setting button(set key and model) in my phone, it only appears on the PC UI.

**Comment by TomLucidor** [comment]
Seconding this but for the config files

**Comment by happyTonakai** [comment]
Need this feature.
